docs/wcs/index.rst

```diff
@@ -49,36 +49,45 @@ The basic workflow is as follows:
 
     4. Use one of the following transformation methods:
 
-       - `~astropy.wcs.wcs.WCS.all_pix2world`: Perform all three
-         transformations (core WCS, SIP and table lookup distortions)
-         from pixel to world coordinates.
+       - From pixels to world coordinates:
 
-       - `~astropy.wcs.wcs.WCS.wcs_pix2world`: Perform just the core
-         WCS transformation from pixel to world coordinates.
+         - `~astropy.wcs.wcs.WCS.all_pix2world`: Perform all three
+           transformations in series (core WCS, SIP and table lookup
```

Comment:
@mdboom I'm not sure if the order is important here, may be change it to: SIP, table lookup, core WCS.